mpmg/src/crawler.py

```diff
@@ -3,60 +3,102 @@
 import sys
 import os
 
-base_url = 'https://transparencia.mpmg.mp.br/db_form/contracheque/'
+base_url = "https://transparencia.mpmg.mp.br/db_form/contracheque/"
 
 # Generate endpoints able to download
 def links_remuneration(month, year):
     links_type = {}
     link = ""
-    if year == "2018" or year == "2019" or (month in ["01", "02", "03", "04"] and year == "2020"):
-        link = base_url + "remuneracao_auxilios_membros_ativos?year=" + year + "&month=" + month + "&format=xlsx&position=membro&status=ativo"
+    if (
```

Comment:
Não seria melhor inverter a condição?

Por exemplo, converter o year e o month para int e depois `if year == 2020 and month > 4` e colocar o trecho do else?